src/Compiler/Microsoft.AspNetCore.Razor.Language/test/CodeGeneration/DesignTimeNodeWriterTest.cs

```diff
@@ -504,7 +504,7 @@ public void LinePragma_Is_Adjusted_On_Windows(string fileName, string expectedFi
             ignoreLineEndingDifferences: true);
     }
 
-    [OSSkipConditionTheory(new[] { "Linux", "osx" })]
+    [OSSkipConditionTheory(new[] { "Linux", "OSX" })]
```

Comment:
As far as I can tell, these tests didn't run with the previous version of xunit, meaning they didn't break when we bumped Roslyn, which brought in the off-by-one error with line pragmas. The skip attribute also should have thrown on the string "osx" which adds further proof.